package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pa11y-ci",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
Should we maybe change this to `2.0.0-alpha` or something, just anything to prevent accidental publishing under 2.0.0